convert2rhel/actions/__init__.py

```diff
@@ -542,17 +542,24 @@ def run_actions():
     return formatted_results
 
 
-def find_failed_actions(results):
+def find_actions_of_severity(results, severity):
     """
-    Process results of run_actions for Actions which abort conversion.
+    Filter results from run_actions() to include only results of ``severity`` or worse.
```

Comment:
Following the example on line #548 and using `:func:`
```suggestion
    Filter results from :func:`run_actions` to include only results of ``severity`` or worse.
```